plansys2_bt_actions/include/plansys2_bt_actions/BTAction.hpp

```diff
@@ -34,9 +34,7 @@ namespace plansys2
 class BTAction : public plansys2::ActionExecutorClient
 {
 public:
-  explicit BTAction(
```

Comment:
I prefer to maintain both constructors for backward compatibility. You can add a warning of deprecation in the old constructor.